### PR TITLE
Update Tendermint definitions

### PR DESCRIPTION
**Pull request by patiee**

### CODE REVIEW

File: codec.proto [hunk 1]
```diff
@@ -9,304 +9,286 @@ import "google/protobuf/descriptor.proto";
...
 
-  SIGNED_MSG_TYPE_UNKNOWN = 0 [(gogoproto.enumvalue_customname) = "UnknownType"];
+  SIGNED_MSG_TYPE_UNKNOWN   = 0 [(gogoproto.enumvalue_customname) = "UnknownType"];
   // Votes
-  SIGNED_MSG_TYPE_PREVOTE = 1 [(gogoproto.enumvalue_customname) = "PrevoteType"];
+  SIGNED_MSG_TYPE_PREVOTE   = 1 [(gogoproto.enumvalue_customname) = "PrevoteType"];
   SIGNED_MSG_TYPE_PRECOMMIT = 2 [(gogoproto.enumvalue_customname) = "PrecommitType"];
   // Proposals
-  SIGNED_MSG_TYPE_PROPOSAL = 3 [(gogoproto.enumvalue_customname) = "ProposalType"];
+  SIGNED_MSG_TYPE_PROPOSAL  = 32 [(gogoproto.enumvalue_customname) = "ProposalType"];
```
Comment (lavabyrd) [inline codec.proto:20]: is this meant to jump to 32? Looks like a typo

File: codec.proto [hunk 2]
```diff
@@ -9,304 +9,286 @@ import "google/protobuf/descriptor.proto";
...
-  EventVote vote_b = 2;
-  int64 total_voting_power = 3;
-  int64 validator_power = 4;
-  Timestamp timestamp = 5 [(gogoproto.nullable) = false, (gogoproto.stdtime) = true];
+// EventList
+message EventList {
+  EventBlock                new_block            = 1;
+  repeated EventTx          transaction         = 2;
+  EventValidatorSetUpdates  validator_set_updates = 3;
 }
```
Comment (lavabyrd) [inline codec.proto:129]: I think it makes sense for this to be at the top (after the enums) vs half way through as this is the overall event list that we use and not just another message

**Comment by lavabyrd** [comment]
I've added some comments and I _think_ it looks ok but we'll need to see how it works out for the other places it gets built (graph_node etc).

Good job on the linting too, can we include the command used to lint in the readme? I know you mentioned you had issues building the protobuf file, did you have to use a different command?

**Review by sosedoff — APPROVED** [review]
LGTM overall, see my comments

File: codec.proto [hunk 3]
```diff
@@ -9,304 +9,279 @@ import "google/protobuf/descriptor.proto";
...
 
 message PublicKey {
   option (gogoproto.compare) = true;
   option (gogoproto.equal) = true;
+
   oneof sum {
-    bytes ed25519 = 1;
+    bytes ed25519   = 1;
     bytes secp256k1 = 2;
-    bytes sr25519 = 3;
```
[the remark this answers is not in the document]
Comment (patiee) [reply codec.proto:214]: Yes, it's not used
https://github.com/figment-networks/tendermint/blob/2b9c0fdc35822e47ee731d52c3c3cd7a9fae3437/proto/tendermint/crypto/keys.proto

**Pull request merged**